cli/schemas/config-file.v1.json

```diff
@@ -404,8 +404,18 @@
       }
     },
     "nodeModulesDir": {
-      "description": "Enables or disables the use of a local node_modules folder for npm packages. Alternatively, use the `--node-modules-dir` flag or override the config via `--node-modules-dir=false`. Requires Deno 1.34 or later.",
-      "type": "boolean"
+      "oneOf": [
+        {
+          "description": "Sets the node_modules management mode for npm packages. Alternatively, use the `--node-modules-dir=<MODE>` flag. Requires Deno 2.0-rc.1 or later.",
+          "default": "none",
+          "enum": ["auto", "manual", "none"]
+        },
+        {
+          "description": "Enables or disables the use of a local node_modules folder for npm packages. Alternatively, use the `--node-modules-dir` flag or override the config via `--node-modules-dir=false`. Requires Deno 1.34 or later.",
+          "type": "boolean",
+          "deprecated": true
```

Comment:
I think maybe we should flip this to `true` after the deno 2.0 release? This will go live immediately after merging.

Reply:
Oh good point